packages/next/server/post-process.ts

```diff
@@ -125,7 +125,7 @@ class FontOptimizerMiddleware implements PostProcessMiddleware {
 
     fontDefinitions.forEach((fontDef) => {
       const [url, nonce] = fontDef
-      const fallBackLinkTag = `<link rel="stylesheet" href="${url}"/>`
+      const fallBackLinkTag = `<link rel="preload" href="${url}" as="style" /> <link rel="stylesheet" href="${url}"/>`
```

Comment:
Adding a `preload` **right before** the actual CSS is completely necessary.

Reply:
:)